File: Unidad3/Java/U3p3.java


package com.mycompany.u3p3;
import java.util.Scanner;
public class U3p3 {

    public static void main(String[] args) {//1
              
        Scanner sc = new Scanner(System.in);//1
        int dia; //2
        System.out.print( "Escribe el numero de la semana numerico");
        dia=sc.nextInt();//4
        switch(dia){
        case 1:
        System.out.println("lunes");
        break;
        case 2:
        System.out.println("Martes");
        break;
        case 3:
        System.out.println("Miercoles");
        break;
        case 4:
        System.out.println("Jueves");
        break;
        case 5:
        System.out.println("Viernes");
        break;
        case 6:
        System.out.println("Sabado");
        break;
        case 7:
        System.out.println("Domingo");
        break;
        default:
            System.out.println("No existe el dia ingresado");
    }
        
        
        
    }
}

File: Unidad3/C#/u3p3.cs
using System;

namespace u3p3

    internal class Program
    {       
            public static void Main(String[] args)
            {
                int dia;
                Console.Write("Escribe el dia de la semana numerico ");
                dia = Convert.ToInt32(Console.ReadLine());
                switch (dia)
                {
                    case 1:
                        Console.WriteLine("Lunes");
                        break;
                    case 2:
                        Console.WriteLine("Martes");
                        break;
                    case 3:
                        Console.WriteLine("Miercoles");
                        break;
                    case 4:
                        Console.WriteLine("Jueves");
                        break;
                    case 5:
                        Console.WriteLine("Viernes");
                        break;
                    case 6:
                        Console.WriteLine("Sabado");
                        break;
                    case 7:
                        Console.WriteLine("Domingo");
                        break;
                      default:
                        Console.WriteLine("No existe el dia ingresado");
                      break;
        }
            }
}   


